### PR TITLE
refactor: Initial color palette changes

### PR DESCRIPTION
**Pull request by BrunoQuaresma**
Screen Shot 2022-07-21 at 10 15 51
Screen Shot 2022-07-21 at 10 14 44

### CODE REVIEW

**Comment by BrunoQuaresma** [comment]
@hkfoster I see the colors are in `hsl`, any specific reasons for that?

File: site/src/components/AvatarData/AvatarData.tsx [hunk 1]
```diff
@@ -48,5 +48,6 @@ const useStyles = makeStyles((theme) => ({
   },
   avatar: {
     marginRight: theme.spacing(1.5),
+    background: "hsl(219, 8%, 52%)",
```
Comment (BrunoQuaresma) [inline site/src/components/AvatarData/AvatarData.tsx:51]: Do you think we could have it as a `palette` object like `theme.palette.paper.dark`? Or something like that?

**Comment by BrunoQuaresma** [comment]
@hkfoster I also see some colors being used directly into the components, ideally IMO, they should be a color in the `palette` object. It could be something simple like `blue.500` `blackAlpha.100` etc. What do you think?

**Comment by BrunoQuaresma** [comment]
Did you use the colors from here?
Screen Shot 2022-07-21 at 10 18 20

**Comment by Kira-Pilot** [comment]
😍 😍 😍 😍 😍 😍 😍 😍 😍 😍 😍 😍 😍 😍 😍 😍 😍 😍 😍 😍 😍 😍 😍 😍 😍 😍 😍 😍 😍 😍 😍 😍

**Comment by Kira-Pilot** [comment]
I left a comment in the snapshots associated with this view, but looks like we lost the darkened box around our code snippet:

Screen Shot 2022-07-21 at 10 13 26 AM

**Comment by Kira-Pilot** [comment]
Just a thought: should we be using a darkened error color? This looks a little bright to me, but maybe it's better that way.
Screen Shot 2022-07-21 at 10 02 18 AM

**Comment by BrunoQuaresma** [comment]
@Kira-Pilot about the Dialog, I think we could keep it as it is for now, and about the dark color in the code example, I will fix that.

**Comment by BrunoQuaresma** [comment]
cc. @Kira-Pilot 
Screen Shot 2022-07-21 at 13 45 43

**Comment by kylecarbs** [comment]
I'm *very* excited for this 😍

**Comment by BrunoQuaresma** [comment]
I forgot to update the copy button color:
Screen Shot 2022-07-21 at 14 31 05

**Review by kylecarbs — APPROVED** [review]

**Pull request merged**